app/src/main/java/com/example/dungeontest/graph/GraphParser.kt

```diff
@@ -1,53 +1,16 @@
 package com.example.dungeontest.graph
 
-import okhttp3.HttpUrl
-import okhttp3.OkHttpClient
-import okhttp3.Request
 import org.jgrapht.Graph
 import org.jgrapht.graph.DefaultEdge
 import org.jgrapht.nio.dot.DOTExporter
 import java.io.StringWriter
 
 class GraphParser {
-//    fun graphToPng(graph: Graph<MapRoom, DefaultEdge>): File {
-//        val dot = graphToDot(graph)
-//        return dotToPng(dot)
-//    }
-    private fun graphToDot(graph: Graph<MapRoom, DefaultEdge>): String {
+    fun graphToDot(graph: Graph<MapRoom, DefaultEdge>): String {
         val exporter = DOTExporter<MapRoom, DefaultEdge>()
         val writer = StringWriter()
 
         return exporter.exportGraph(graph, writer).toString()
```

Comment:
I expressed confusion about this in the last PR. after looking more closely, I think this is currently compiling because kotlin returns this trivial singleton object 'Unit' from functions that java would call 'void', and the Unit object has a (useless) toString implementation:
![image](https://github.com/BareBeaverBat/CSE5236-SU24-DungeonMapDigitizer/assets/54820763/e0166d3e-8a84-4287-b49f-0093a0363bc2)
```suggestion
        exporter.exportGraph(graph, writer)
        return writer.toString()
```